### PR TITLE
PMM-13037 Restructured_TOC_PMM

### PR DESCRIPTION
**Pull request by rasika-chivate**
Created a new structure for PMM docs.

### CODE REVIEW

File: docs/install-pmm/install-pmm-client/docker.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    -e PMM_AGENT_SERVER_INSECURE_TLS=1 \
+    -e PMM_AGENT_SETUP=1 \
+    -e PMM_AGENT_CONFIG_FILE=config/pmm-agent.yaml \
+    --volumes-from pmm-client-data \
+    percona/pmm-client:2
+    ```
+!!! hint alert-success "Tips"
+    You can find a complete list of compatible environment variables [here](../../details/commands/pmm-agent.md).
+
+4. Check status.
```
Comment (alina-derkach-oaza) [inline docs/install-pmm/install-pmm-client/docker.md:41]: The list is broken

File: docs/install-pmm/install-pmm-client/connect-database/mongodb.md [hunk 2]
```diff
@@ -24,74 +24,62 @@ Run the example codes below in a `mongo` session to:
...
-```{.javascript data-prompt=">"}
- db.getSiblingDB("admin").createRole({
-    "role": "pbmAnyAction",
-    "privileges": [{
-        "resource": {
-            "anyResource": true
-        },
-        "actions": [
-            "anyAction"
+=== "### Create roles with privileges for backups and QAN"
```
Comment (alina-derkach-oaza) [inline docs/install-pmm/install-pmm-client/connect-database/mongodb.md:27]: remove extra ###

File: docs/troubleshoot/dbaas/operator_issues.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+Once OLM is installed, PMM does the following actions to install each operator:
+{.power-number}
+
+1. Creates a subscription for an operator
+2. Approves the first available install plan automatically
+
+Once the install plan is approved OLM will create a corresponding ClusterServiceVersion automatically and install the operator.
+
+During this process the following steps might go wrong
```
Comment (alina-derkach-oaza) [inline docs/troubleshoot/dbaas/operator_issues.md:11]:
```suggestion
During this process the following steps might go wrong:
{.power-number}
```

File: docs/reference/ui/ui_components.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ The main menu is part of the Grafana framework and is visible on every page.
 
 | Item (Top)                         | Name                 | Description
 |:----------------------------------:|----------------------|-------------------------------
-| ![!image](../_images/pmm-logo.png) | Home                 | Link to home dashboard.
+| {{icon.percona}}                   | Home                 | Link to home dashboard.
```
Comment (alina-derkach-oaza) [inline docs/reference/ui/ui_components.md:18]: All images in this doc are broken

File: docs/reference/third-party/postgresql.md [hunk 5]
```diff
@@ -30,17 +30,21 @@ To use grafana with external PostgreSQL add `GF_DATABASE_*` environment variable
...
+
+    ```
+    /pmm-server-certificates# la -la
+    drwxr-xr-x 1 root    root    4096 Apr  5 12:43 .
+    drwxr-xr-x 1 root    root    4096 Apr  5 12:43 ..
+    -rw------- 1 grafana grafana 1391 Apr  5 12:38 certificate_authority.crt
+    -rw------- 1 grafana grafana 1257 Apr  5 12:38 pmm_server.crt
+    -rw------- 1 grafana grafana 1708 Apr  5 12:38 pmm_server.key
+    ```
 3. Attach `pg_hba.conf` and certificates to the PostgreSQL image.
```
Comment (alina-derkach-oaza) [inline docs/reference/third-party/postgresql.md:48]: The list is broken.

File: docs/install-pmm/HA.md [hunk 6]
```diff
@@ -303,6 +309,7 @@ To set up PostgreSQL:
 ### **Step 6: Running PMM Services**
```
Comment (alina-derkach-oaza) [inline docs/install-pmm/HA.md:309]: Add {.power-number} to Step 7.

File: docs/install-pmm/install-pmm-server/baremetal/docker/index.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+    !!! summary alert alert-info ""
+        - Pull the Docker image.
+        - Copy it to create a persistent data container.
+        - Run the image.
+        - Open the PMM UI in a browser.
+
+    ---
+
+You can store data from PMM in:
```
Comment (alina-derkach-oaza) [inline docs/install-pmm/install-pmm-server/baremetal/docker/index.md:29]:
```suggestion
You can store data from PMM in:
{.power-number}
```

File: docs/install-pmm/install-pmm-server/baremetal/virtual/login_UI.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+    5. Click **Apply SSH Key**. (This copies the public key to `/home/admin/.ssh/authorized_keys` in the guest).
+
+    6. Log in via SSH (`N.N.N.N` is the guest IP address).
+
+        ```sh
+        ssh -i admin admin@N.N.N.N
+        ```
+
+??? info "(Optional) Set up static IP via CLI"
```
Comment (alina-derkach-oaza) [inline docs/install-pmm/install-pmm-server/baremetal/virtual/login_UI.md:62]:
```suggestion
??? info "(optional) Set up static IP via CLI"
```

File: docs/install-pmm/install-pmm-client/package_manager.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+
+!!! hint alert alert-success "Tip"
+    If you have used `percona-release` before, disable and re-enable the repository:
+
+    ```sh
+    percona-release disable all
+    percona-release enable original release
+    ```
+
+=== "Debian-based"
```
Comment (alina-derkach-oaza) [inline docs/install-pmm/install-pmm-client/package_manager.md:13]: You have different types of lists in this doc

File: docs/pmm-upgrade/upgrade_agent.md [hunk 10]
```diff
@@ -0,0 +1,10 @@
+# Upgrade PMM agent
+
+PMM-Agent can be updated from tarball:
```
Comment (alina-derkach-oaza) [inline docs/pmm-upgrade/upgrade_agent.md:3]:
```suggestion
PMM-Agent can be updated from tarball:
{.power-number}
```

File: mkdocs-base.yml [hunk 11]
```diff
@@ -153,254 +156,378 @@ extra:
...
+      - troubleshoot/upgrade_issues.md
+      - troubleshoot/config_issues.md
+      - DBaaS issues:
+        - troubleshoot/dbaas/index.md      
+        - troubleshoot/dbaas/olm_issues.md
+        - troubleshoot/dbaas/operator_issues.md
+      - troubleshoot/alerting_issues.md
+      - troubleshoot/qan_issues.md
+      - troubleshoot/plugin_issues.md
+  - Release Notes:
```
Comment (alina-derkach-oaza) [inline mkdocs-base.yml:447]:
```suggestion
  - Release notes:
```

File: docs/install-pmm/install-pmm-client/connect-database/mongodb.md (hunk 2 above)
Comment (rasika-chivate) [reply docs/install-pmm/install-pmm-client/connect-database/mongodb.md:27]: Fixed

**Comment by ademidoff** [comment]
Closing as stale.

**Pull request closed without merging**